common/djangoapps/util/password_policy_validators.py

```diff
@@ -320,3 +320,11 @@ def _validate_password_dictionary(value):
             if edit_distance <= password_max_edit_distance:
                 raise ValidationError(_("Password is too similar to a dictionary word."),
                                       code="dictionary_word")
+
+
+def normalize_password(password):
+    """
+    Normalize all passwords to 'NFKC' across the platform to prevent mismatched hash strings when comparing entered
+    passwords on login. See LEARNER-4283 for more context.
```

Comment:
Nice comment. I often don't bother sprinkling links to old tickets in the codebase, but that ticket has so much context in it, probably a good idea.